go.mod

```diff
@@ -5,6 +5,7 @@ go 1.19
 require (
 	cloud.google.com/go/compute/metadata v0.2.3
 	github.com/google/go-cmp v0.5.9
+	github.com/google/s2a-go v0.1.0
```

Comment:
this is unfortunate... it adds more transitive dependencies on appengine and means more consumers of this module now link in appengine code - xref https://github.com/kubernetes/kubernetes/pull/120341#discussion_r1312959575

Reply:
can use of this be isolated in a different package, and/or build-tagged so all consumers are not exposed to an experimental / unstable transitive dependency?

Reply:
For example, https://github.com/google/s2a-go/blob/3d294feadf626dd327a2cbba9e91d913aefbdea3/internal/handshaker/service/service.go#L31

Reply:
We should be able to drop the appengine depdency from s2a-go repo.

Reply:
synced with @liggitt 
removing appengine dependency from s2a in https://github.com/google/s2a-go/pull/119